src/kernels/composable_kernel/include/utility/amd_buffer_addressing.hpp

```diff
@@ -139,12 +139,7 @@ __llvm_amdgcn_buffer_store_bf16x4(ushort4_t vdata,
                                   bool slc) __asm("llvm.amdgcn.buffer.store.v4bf16");
 
 #if CK_USE_AMD_BUFFER_ATOMIC_FADD
-#if CK_HIP_VERSION_FLAT >= 3010020405
```

Comment:
Please look at question at https://github.com/ROCmSoftwarePlatform/MIOpen/issues/650#issuecomment-757013445. Perhaps we need to use the current version from the staging.

Reply:
fixed